cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -414,6 +413,9 @@ def add_new_case(
         logger.info(
             f"Adding docket for {case_name}: {citation.base_citation()}"
         )
+        if not Court.objects.filter(id=court_id).exists():
+            logger.warning(f"No court found for ID: {court_id}")
```

Comment:
Is a warning enough here? Don't you need it to crash or `return` or something? I assume line 424 would fail if the court didn't exist, right?

Reply:
I do.

Reply:
im trying to avoid a crash - and generate a log so that I can process NY - find all the courts that need to be added and then reprocess the ny reporter once again.

Reply:
thanks for catching this.